### PR TITLE
Added Contributing.md file

### PR DESCRIPTION
**Pull request by ManoBharathi93**

### CODE REVIEW

**Comment by dschonholtz** [comment]
@Torantulino can we make this fairly high priority, but also add in contribution guidelines on what kind of code styling/basic standards we want?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+```
+git clone https://github.com/Torantulino/Auto-GPT
+```
+3. Create a new branch for your changes using the following command:
+
+```
+git checkout -b "branch-name"
+```
+4. Make your changes to the code or documentation.
```
Comment (Torantulino) [inline CONTRIBUTING.md:16]:
```suggestion
4. Make your changes to the code or documentation. 
**Ensure each commit is small and addresses a SINGLE issue.** PRs with commits that sneak in extra tweaks will likely be rejected.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Example: Improve User Interface or Add Documentation.
+
+5. Add the changes to the staging area using the following command:
+```
+git add . 
+```
+
```
Comment (Torantulino) [inline CONTRIBUTING.md:23]:
```suggestion
Or, if using Git in VS-Code, select specific lines of code and commit them.

```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+git push origin branch-name
+```
+8. Go to the GitHub website and navigate to your forked repository.
+
+9. Click the "New pull request" button.
+
+10. Select the branch you just pushed to and the branch you want to merge into on the original repository.
+
+11. Add a description of your changes and click the "Create pull request" button.
+
```
Comment (Torantulino) [inline CONTRIBUTING.md:39]:
```suggestion
- Be as clear, descriptive and specific as possible.
- Use markdown formatting for a clear and well written post.

```

**Review by Torantulino — APPROVED** [review]

**Pull request merged**